pkg/runtime-tools/generate/generate.go

```diff
@@ -309,6 +310,13 @@ func (g *Generator) AdjustCgroupsPath(path string) {
 	}
 }
 
+// AdjustOomScoreAdj adjusts the cgroup pseudofs path in the OCI Spec.
```

Comment:
Please fix this comment. It could be something like:
```
// AdjustOomScoreAdj adjusts the kernel's Out-Of-Memory (OOM) killer score for the container
```

I don't know if most folks are generally aware that the kubelet also actively sets/adjusts the OOM Score adjustment field for containers based on the QoS class (and IIRC memory requests for Burstable containers). If not, maybe also a word of warning here warranted. @samuelkarp WDYT ?

Reply:
We extended the k8s qos classes to support more workloads, in our implementation, we will override the Kubelet's value when k8s qos class is Burstable, and respect Guaranteed and BestEffort qos classes.